improver_tests/blending/test_utilities.py

```diff
@@ -64,17 +70,81 @@ def input_cube_fixture():
     return cycle_cubes.merge_cube()
 
 
+@pytest.fixture(name="model_cube")
+def model_cube_fixture():
+    """Set up a cube for model blending"""
+    thresholds = [10, 20]
+    data = np.ones((2, 2, 2), dtype=np.float32)
+    model_ids = [0, 1000]
+    model_names = ["uk_det", "uk_ens"]
+    model_cubes = iris.cube.CubeList([])
+    for id, name in zip(model_ids, model_names):
+        model_id_coord = iris.coords.AuxCoord([id], long_name=MODEL_BLEND_COORD)
+        model_name_coord = iris.coords.AuxCoord([name], long_name=MODEL_NAME_COORD)
+        model_cubes.append(
+            set_up_probability_cube(
+                data,
+                thresholds,
+                spatial_grid="equalarea",
+                time=datetime(2017, 11, 10, 4),
+                frt=datetime(2017, 11, 10, 1),
+                include_scalar_coords=[model_id_coord, model_name_coord],
+            )
+        )
+    return model_cubes.merge_cube()
+
+
 @pytest.mark.parametrize(
     "input_coord_name", ("forecast_reference_time", "forecast_period")
 )
-def test_find_blend_dim_coord_noop(input_cube, input_coord_name):
+def test_find_blend_dim_coord_noop(cycle_cube, input_coord_name):
```

Comment:
What does ```noop``` stand for?

Reply:
"no operation" - it means in this case that we gave the function an input that is already a dimension coord, so it won't change it.  This is a pretty standard shorthand I think.